src/components/Books.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import BookList from './BookList';
+
+function Books() {
+  const [books, setBooks] = useState([
+    {
+      title: 'Fantastic Beasts And Where To find them',
+      author: 'J. K. Rowling',
+      category: 'action',
+      chapter: 'Chapter 17',
+    },
+    {
+      title: 'The Hunger games',
+      author: 'Suzanne Collins',
+      category: 'action',
+      chapter: 'Chapter 3',
+    },
+    {
+      title: 'Jurassic Parc',
+      author: 'Michael Crichton',
+      category: 'sci-fi',
+      chapter: 'Introduction',
+    },
+  ]);
+  const handleDelete = (title) => {
+    const newBookList = books.filter((book) => book.title !== title);
+    setBooks(newBookList);
+  };
+  return (
+    <div>
+      <BookList books={books} handleDelete={handleDelete} />
+      <div style={{
+        width: '100%',
+        background: 'black',
+        height: '2px',
+        marginTop: '10px',
+        marginBottom: '10px',
+      }}
+      />
+      <form>
+        <h1>ADD NEW BOOK</h1>
+        <div>
+          <input type="text" />
+          <select>
+            <option value="action" selected>Action</option>
+            <option value="sci-fi">Sci-Fi</option>
+            <option value="Economy">Economy</option>
+          </select>
```

Comment:
- [ ] Well done :clap: you did a very nice work :+1: but if you read the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) there they mentioned you will remove the category and replace it with author field so kindly fix this issue :slightly_smiling_face: 

> **IMPORTANT** In the design you can see an input for a category - please replace it with an author.